File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("multifon-client_sdk")

describe("MultifonClientSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// MultifonClient SDK exists test

require_once __DIR__ . '/../multifonclient_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = MultifonClientSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}
